app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -77,7 +72,7 @@ def submit_configure_imported_vm_networks(userid, vm_id)
   def check_import_supported!(source_provider)
     raise _('Cannot import archived VMs') if source_provider.nil?
 
-    raise _('Cannot import to a RHEV provider of version < 4.1.5') unless validate_import_vm
+    raise _('Cannot import to a RHEV provider of version < 4.1.5') unless supports_vm_import?
```

Comment:
I believe this should be: unsupported_reason(:vm_import) unless supports_vm_import?

Reply:
Done.